### DISCUSSION TITLE
Colors (Greys) of the components

### DISCUSSION THREAD

**Discussion by lathoub** [rrainey/virtual-agc-mechanical-cad#7] [Q&A]
Hi all,

I'm 3D printing the components of the DSKY and was wondering if someone has the color values of them?
Everything looks grey, but want to use the right shade of grey :-)

Thanks!
Lathoub

**Comment by rrainey** [comment]
The original drawing sheets will include Federal Standard 595 color specifications for applicable parts.  In all sheets I have inspected closely, the gray is specified as FS 36231 "Dark Gull Gray".   I mentioned this in passing on [this page](https://github.com/rrainey/DSKY-alarm-panel-replica).

**Reply by lathoub** [reply]
Thanks @rrainey .
Looking at this [image](https://github.com/rrainey/agc-mechanical-cad/blob/master/images/ASSEMBLED%202.png), the front looks darker than FS 36231. This [image ](https://www.rrauction.com/auctions/lot-detail/341744505692223--apollo-display-and-keyboard-assembly-dsky-/?cat=0) has the front in what looks like FS 36231, but the back components as black (with a gasket in white). Maybe they were all different? (or were they different in the CM and LM?)

**Reply by rrainey** [reply]
I'd be surprised if there were undocumented variations in flight hardware.  And - so far as I know - the CM and LM paint specifications were the same. If you are looking for how to recreate an accurate version of FS 36231 -- short of buying a pre-mixed commercial paint -- you might have a look at [this site](http://www.perbang.dk/rgb/6F7571/).  If you are suspecting that the actual color used was not 36231, you'd have to put a color card up to an actual artifact while keeping in mind that the artifacts are now 60+ years old.

**Reply by rrainey** [reply]
For what it is worth, this topic had been on my mind a few months back too.  

I purchased some Testors Dark Gull Gray (4755) enamel and painted a few parts with it.  The results seemed much too dark. But it is difficult to trust ones eyes here, so I did further research.  There's a local supplier for FED-STD color mixing, [LVP Powder and Paints](https://www.lvppaints.com/). I ordered a few 12 oz. cans of FS 36231 from them.  I painted a sample board and took a photo alongside a color calibration card.  Here is a white and black level calibrated version of the image (sample board at the bottom):

<img width="326" alt="FS36231-sample" src="https://user-images.githubusercontent.com/271126/227721896-5e5d6807-588e-42ea-98f1-e79da4878e34.png">

This gray level seems much more consistent with  60's era DSKY photos. This shade is considerably lighter than Testors' version.  My conclusion: don't rely on traditional scale modeling paints to get an accurate FED-STD color match. I plan to use this LVP paint on my projects.

**Comment by lathoub** [comment]
I was to use [MrPaint](https://spruebrothers.com/mrp100-mrp-mr-paint-dark-gull-grey-us-nato-interior-fs36231-30ml-for-airbrush-only/) - they normally are very accurate. However its look darker than the grey on the images.

I have a color card and will try to match the color with my ref images - and report back.

**Reply by lathoub** [reply]
My color card fan deck indicates 36231 to be pretty close to the sample from LVP.
Some web images show a _darker_ grey ([LM15](https://commons.wikimedia.org/wiki/File:Interior_of_Apollo_15_lunar_module_%28prior_to_launch%29.jpg) [CM](https://airandspace.si.edu/explore-and-learn/multimedia/detail.cfm?id=3055) [LM](https://airandspace.si.edu/multimedia-gallery/99-15227hjpg) [LM11](https://newatlas.com/apollo-guidance-computer-dsky-space-memorabilia/49631/))

The item on display [here](https://newatlas.com/apollo-guidance-computer-dsky-space-memorabilia/49631/) is very faded after years of conservation.

So I will paint my front using 36231, and add some lighter patches using a shading technique

**Comment by rrainey** [comment]
I was offered the opportunity to visit the Apollo 7 Command Module exhibit at the Frontiers of Flight Museum here in Dallas. I was permitted to take a few photos with both a color calibration card and a sample board of FS36231 (paint source mentioned above) next to the main panel, which is the same DSKY color.  Both photos were take with an iPhone. The photo with the color card was white/black calibrated using Photoshop (Image > Adjustments > Curve ...). The sample board photo has no post processing applied.  

![Apollo7-colorcard-small](https://user-images.githubusercontent.com/271126/230376614-34a7ce62-84a0-4e95-9da8-453b66f2dd14.png)

Keep in mind that the paint inside the Apollo 7 CM has had close to sixty years to age, so make of that whatever you like.

![Apollo7-fs36231-small](https://user-images.githubusercontent.com/271126/230377003-b4479c94-8c13-4a82-b72d-2cf959255c75.png)

**Reply by rrainey** [reply]
It is worth pointing out that this is the same sample board appearing in my earlier post in this thread.

**Reply by lathoub** [reply]
Oh wow - great! That settles it then for FS36231
